test/expected.txt

```diff
@@ -297,6 +299,10 @@
 || 
 || 0 persons listed!
 || ===================================================
+|| Enter command: || [Command entered:  repeat]
+|| 
+|| 0 persons listed!
```

Comment:
Good to see that its working. however, for a command like this, usually needs a thorough testing to make sure it works for all other commands.